### PR TITLE
[ernie-relative] add OpMapper for paddle expand and expand_v2

### PR DESCRIPTION
**Pull request by Aganlengzi**
Add OpMapper for paddle expand, expand_v2 and related unittests referring to https://github.com/PaddlePaddle/CINN/pull/961, for expand, CINN only supports static attribute expand_times depending on https://github.com/PaddlePaddle/Paddle/pull/46776.

### CODE REVIEW

**Review by wzzju — APPROVED** [review]
LGTM

File: cinn/frontend/op_mappers/paddle/CMakeLists.txt [hunk 1]
```diff
@@ -22,6 +22,8 @@ gather_srcs(cinnapi_src SRCS
     constant.cc
     where.cc
     squeeze.cc
-    layer_norm.cc)
+    layer_norm.cc
+    expand.cc)
 
-cc_test(test_layer_norm SRCS layer_norm_test.cc DEPS cinncore)
+cc_test(test_paddle_layer_norm SRCS layer_norm_test.cc DEPS cinncore)
```
Comment (thisjiang) [inline cinn/frontend/op_mappers/paddle/CMakeLists.txt:28]: 在这个PR上我加上了op_mapper的测试机制：https://github.com/PaddlePaddle/CINN/pull/975 ，以后只需在`python/tests/op_mappers`目录下添加对应的python单测就行了。

这种C++ 测试等PR975 合入了我都删了吧，都改为python单测

**Review by thisjiang — APPROVED** [review]
LGTM

File: cinn/frontend/op_mappers/paddle/CMakeLists.txt (hunk 1 above)
Comment (Aganlengzi) [reply cinn/frontend/op_mappers/paddle/CMakeLists.txt:28]: 有计划为我加的paddle的OpMapper都加下UT，后面我看情况新增python吧。

**Pull request merged**